packages/xl-ai/src/components/AIMenu/AIMenu.tsx

```diff
@@ -82,6 +85,36 @@ export const AIMenu = (props: {
     }
   }, [aiResponseStatus]);
 
+  const placeholder = useMemo(() => {
+    if (aiResponseStatus === "thinking") {
+      return dict.formatting_toolbar.ai.thinking;
+    } else if (aiResponseStatus === "ai-writing") {
+      return dict.formatting_toolbar.ai.editing;
+    } else if (aiResponseStatus === "error") {
+      return dict.formatting_toolbar.ai.error;
+    }
+
+    return dict.formatting_toolbar.ai.input_placeholder;
+  }, [aiResponseStatus, dict]);
+
+  const rightSection = useMemo(() => {
+    if (aiResponseStatus === "thinking" || aiResponseStatus === "ai-writing") {
+      return (
+        <Components.SuggestionMenu.Loader
+          className={"bn-suggestion-menu-loader bn-combobox-right-section"}
+        />
+      );
+    } else if (aiResponseStatus === "error") {
+      return (
+        <div className={"bn-combobox-right-section bn-combobox-error"}>
+          <span>!</span>
```

Comment:
this ! is selectable, which is a bit weird. Also, maybe there are nicer icons for this?